asp.tf

```diff
@@ -1,15 +1,16 @@
-locals {
-  ase_name = "core-compute-${var.env}"
-}
-
-module "appServicePlan" {
-  source              = "git@github.com:hmcts/cnp-module-app-service-plan?ref=master"
-  location            = "${var.location}"
-  env                 = "${var.env}"
-  resource_group_name = "${azurerm_resource_group.rg.name}"
-  asp_sku_size        = "${var.asp_sku_size}"
-  asp_capacity        = "${var.asp_capacity}"
-  asp_name            = "${var.product}"
-  ase_name            = "${local.ase_name}"
-  tag_list            = "${var.common_tags}"
-}
+//// commented out to stop recreating legacy app service plans
+//locals {
+//  ase_name = "core-compute-${var.env}"
+//}
+//
+//module "appServicePlan" {
+//  source              = "git@github.com:hmcts/cnp-module-app-service-plan?ref=master"
+//  location            = "${var.location}"
+//  env                 = "${var.env}"
+//  resource_group_name = "${azurerm_resource_group.rg.name}"
+//  asp_sku_size        = "${var.asp_sku_size}"
+//  asp_capacity        = "${var.asp_capacity}"
+//  asp_name            = "${var.product}"
+//  ase_name            = "${local.ase_name}"
+//  tag_list            = "${var.common_tags}"
+//}
```

Comment:
Do we still needed it for FE apps?

Reply:
@timja?

Reply:
it's only needed to be created once, and it's only needed in AAT and Demo,

it's unlikely the existing ones will need recreating, and if they do this can be uncommented, but leaving it in risks the prod ones being re-created which is expensive